libs/dh/market-participant/feature-gridarea/src/lib/details-header/dh-market-participant-gridarea-details-header.component.html

```diff
@@ -18,21 +18,52 @@
 <ng-container
   *transloco="let t; read: 'marketParticipant.gridAreas.detailsHeader'"
 >
-  <div>
+  <ng-container *ngIf="gridArea">
     <div class="header-grid-area-code">
-      {{ t("code") + " " + gridArea?.code }}
+      {{ t("code") + " " + gridArea.code }}
     </div>
     <h1 class="header-grid-area-name">
-      {{ gridArea?.name }}
+      {{ gridArea.name }}
     </h1>
     <div class="header-grid-area-access-provider">
       {{
-        (gridArea?.actorName | dhEmDashFallback) +
+        (gridArea.actorName | dhEmDashFallback) +
           " | " +
           t("actorId") +
           " " +
-          (gridArea?.actorNumber | dhEmDashFallback)
+          (gridArea.actorNumber | dhEmDashFallback)
       }}
     </div>
-  </div>
+
+    <mat-divider></mat-divider>
+
+    <div
+      class="grid-area-property-container"
+      *transloco="let topTrans; read: 'marketParticipant.gridAreas'"
+    >
+      <ng-container
+        *ngFor="
+          let item of [
+            {
+              header: t('priceAreaCode'),
+              value: topTrans('priceAreaCodes.' + gridArea.priceAreaCode)
+            },
+            {
+              header: t('lossMeteringPoint'),
+              value: undefined | dhEmDashFallback
+            },
+            {
+              header: t('correctionMeteringPoint'),
+              value: undefined | dhEmDashFallback
+            }
+          ]
```

Comment:
question: is this "final" or will the static array definition be replaced with something more dynamic?

Reply:
More entries will be added in future PRs, but the array will continue be static.

Reply:
@ManBearTM

Reply:
@defectiveAi Bjørn has today and tomorrow off. I can help with the PR review instead.

> More entries will be added in future PRs, but the array will continue to be static.

In this case let's remove the array (and the loop) and have each section as a separate HTML elements. There's going to be some repetition, but we'll avoid the above logic in the template.

Reply:
Fixed